### ISSUE TITLE
关于 positive risk 和 negative risk 的计算

### ISSUE THREAD

**Issue by FractureSR** [wangqr/nnpu#2]
您好，关于您在 loss.py 中对 positive risk 和 negative risk 的实现
def positive_risk(y_true, y_pred):
    global pi_p

    loss_func = tf.nn.sigmoid

    n_positive = tf.maximum(1., tf.reduce_sum(y_true))
    r_plus = loss_func(-y_pred)
    return tf.reduce_sum(pi_p * y_true / n_positive * r_plus)

def negative_risk(y_true, y_pred):
    global pi_p

    loss_func = tf.nn.sigmoid

    n_positive = tf.maximum(1., tf.reduce_sum(y_true))
    n_unlabeled = tf.maximum(1., tf.reduce_sum(1 - y_true))
    r_minus = loss_func(y_pred)
    return tf.reduce_sum(
        ((1 - y_true) / n_unlabeled - pi_p * y_true / n_positive) * r_minus)

对比您的实现和论文原实现，r_plus 和 r_minus 计算时 y_pred 的符号是否写反了？即计算 positive_risk 时使用 y_pred，计算 negative_risk 时使用 -y_pred。
期待您的答复。

**Comment by FractureSR** [comment]
我已经发现是我对原实现的代码理解有误，您是正确的，谢谢。
